ws.nimble

```diff
@@ -4,12 +4,12 @@ author = "Status Research & Development GmbH"
 description = "WS protocol implementation"
 license = "MIT"
 
-requires "nim >= 1.2.6"
-requires "chronos >= 2.5.2 & < 3.0.0"
+requires "nim >= 1.4.0"
```

Comment:
we should stick with 1.2.6 for now, until we've adopted 1.4.x across the board.

Reply:
Done